Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -61,10 +76,69 @@ - (instancetype)initWithBridge:(nonnull RCTBridge *)bridge
     _animationNodes = [NSMutableDictionary new];
     _eventDrivers = [NSMutableDictionary new];
     _activeAnimations = [NSMutableSet new];
+    _uiManagerOperationQueue = [NSMutableArray new];
+    _layoutProps = @[
```

Comment:
I'm wondering if I can extend this set. E.g. in my app (cc. @christianbaroni, @jinchung) I wish to animate `blurAmount` from 'react-native-blur-view`. I managed to do it with reanimated, bc we exported the method for handling this.

Reply:
Basically I'd love to have sth like `configureProps` 😄

Reply:
We're discussing this - all properties should be animated automatically, but a method for adding extra props that needs to go through the `enqueueUpdateViewOnUIManager` should be available. Will try to add this.

Reply:
Thanks!

Reply:
Added addEnqueuedUpdateProp method :)

Reply:
Just to clarify this, we want to use `synchronouslyUpdateViewOnUIThread` when possible since it will result in the best UX / performance and fallback to `enqueueUpdateViewOnUIManager` for props that cannot be updated synchronously from the UI thread because they require to go through the shadow tree (usually because of layout).